QUEST_JOBSTEP.tsv

```diff
@@ -2095,46 +2095,46 @@ QUEST_JOBSTEP_20150317_002094	Necromancer told you to bring the corpses of the m
-QUEST_JOBSTEP_20150317_002115	Defeated Poatas. Report to Barbarian Master.
-QUEST_JOBSTEP_20150317_002116	Go meet Rodelero Master in Fedimian.
-QUEST_JOBSTEP_20150317_002117	Rodelero Master wants to accumulate real experiences. Defeat Collimencia in Forest of Incremental Rocks to prove your skills.
-QUEST_JOBSTEP_20150317_002118	Defeated Colimencia. Report to Rodeleo Master.
-QUEST_JOBSTEP_20150317_002119	Go meet Cataphract Master in High Crooked Road.
-QUEST_JOBSTEP_20150317_002120	Cataphract Master told you that it is important to break anything that disturbes your way. Defeat Gaigalas in Flowerless Forest to prove your skills.
-QUEST_JOBSTEP_20150317_002121	Defeated Gaigalas. Report to Cataphract Master.
-QUEST_JOBSTEP_20150317_002122	Go to Squire Master in Fedimian.
-QUEST_JOBSTEP_20150317_002123	Defeate Poatas and find the seal of family
-QUEST_JOBSTEP_20150317_002124	Squire Master has the symbol of Legwyn Family. Defeat Poatas in Forest of Incremental Rocks to find the symbol.
-QUEST_JOBSTEP_20150317_002125	Report to Squire Master
-QUEST_JOBSTEP_20150317_002126	Found the family's ornament. Report to Squire Master.
-QUEST_JOBSTEP_20150317_002127	Get the ornament of Legwyin Family
-QUEST_JOBSTEP_20150317_002128	Talk to Corsair Master
-QUEST_JOBSTEP_20150317_002129	Go meet Squire Master in Starving Demon's Road
-QUEST_JOBSTEP_20150317_002130	Cursoir Master wants you to prove your skills. Defeat Honeypins in Deep Indigo Forest to prove your skills.
-QUEST_JOBSTEP_20150317_002131	Report to Corsair Master
-QUEST_JOBSTEP_20150317_002132	Defeated Honeypins. Report to Corsair Master.
-QUEST_JOBSTEP_20150317_002133	Talk to Doppelsoeldner Master
-QUEST_JOBSTEP_20150317_002134	Go meet Doppelsoeldner Master in Forgotten Port.
-QUEST_JOBSTEP_20150317_002135	Doppelsoeldner Master wants you to prove your skills by defeating Gaigalas in Flowerless Forest.
-QUEST_JOBSTEP_20150317_002136	Report to Doppelsoeldner Master
-QUEST_JOBSTEP_20150317_002137	Defeated Gaigalas. Report to Doppelsoeldner Master.
+QUEST_JOBSTEP_20150317_002107	Go find the Highlander Master in Klaipeda.
+QUEST_JOBSTEP_20150317_002108	The Highlander Master wants you to gather more experience. He asks you to defeat Honeypins at Deep Indigo Forest. 
+QUEST_JOBSTEP_20150317_002109	You defeated enough Honeypins. Report back to the Highlander Master.
+QUEST_JOBSTEP_20150317_002110	The Hoplite Master possesses the skill which his colleagues can rely on. Defeat Collimencias in Forest of Incremental Rocks to prove your skills.
+QUEST_JOBSTEP_20150317_002111	You defeated enough Colimencias. Report back to the Hoplite Master.
+QUEST_JOBSTEP_20150317_002112	Go to the Barbarian Master in Siauliai Eastern Woods.
+QUEST_JOBSTEP_20150317_002113	Meet the Barbarian Master in Siauliai Eastern Woods
+QUEST_JOBSTEP_20150317_002114	The Barbarian Master wants other people to respect him for defeating Poatas that are known for attacking people. Defeat Poatas in Forest of Incremental Rocks.
```

Comment:
The Barbarian Master wants other people to respect him for defeating Poatas -> The Barbarian Master wants other people to respect him by defeating the Poata

Defeat Poatas -> Defeat Poata